### PR TITLE
2509 document featsink set default sink decouple = true for all sink

### LINKED ISSUE #2509: Document: feat(sink): set default_sink_decouple = true for all sink
This issue tracks the documentation update needed for the merged PR #18182.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/18182
Source PR Merged At: 2024-08-26T04:37:39Z

If it is a major improvement that deserves a new page or a new section in the documentation, please check if we should label it as an experiment feature.

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Description

- Replace the sink list in the sink_decoupling page with one sentence "All sinks enable this feature by default".
- Align the parameter description of [commit_checkpoint_interval](https://docs.risingwave.com/docs/next/sink-to-starrocks/#parameters) for 4 sinks whose default `commit_checkpoint_interval` value is 1.

## Related code PR

https://github.com/risingwavelabs/risingwave/pull/18182

## Related doc issue

Resolves https://github.com/risingwavelabs/risingwave-docs/issues/2509

## Checklist

- [ ] I have checked the doc site preview, and the updated parts look good.
- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/guides/sink-to-clickhouse.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ WITH (
 | `clickhouse.password`   | Required. Password for accessing the ClickHouse server.|
 | `clickhouse.database`  | Required. Name of the ClickHouse database that you want to sink data to.|
 | `clickhouse.table`      | Required. Name of the ClickHouse table that you want to sink data to.|
-| `commit_checkpoint_interval`| Optional. Commit every N checkpoints (N > 0). If not set, it commits to ClickHouse at every checkpoint.|
+| `commit_checkpoint_interval`| Optional. You can use this parameter to decouple the downstream system’s commit from RisingWave’s commit. This means that instead of committing data to the downstream system at every barrier, RisingWave will commit data only when the specified checkpoint interval is reached. For instance, if `commit_checkpoint_interval` is set to `5`, RisingWave will commit data every five checkpoints. The default value is `1`. Note that when `commit_checkpoint_interval` is a positive integer larger than `1`, the [`sink_decouple`](/data-delivery.md#sink-decoupling) option will be enabled automatically.|
```
Comment (xxhZs) [inline docs/guides/sink-to-clickhouse.md]: all commit_checkpoint_interval need to modify:
pr is https://github.com/risingwavelabs/risingwave/pull/18348

Comment (WanYixian) [reply docs/guides/sink-to-clickhouse.md]: Update for all sinks that contain `commit_checkpoint_interval` (ClickHouse, Deltalake, Iceberg, Starrocks), please take another look

Comment (xxhZs) [reply docs/guides/sink-to-clickhouse.md]: `The default value is `1`. Note that when `commit_checkpoint_interval` is a positive integer larger than `1`, the [`sink_decouple`](/data-delivery.md#sink-decoupling) option will be enabled automatically.|`

if sink_decouple = true/defalut default commit_checkpoint_interval = 10;
if sink_decouple = false default commit_checkpoint_interval = 1;
if sink_decouple = false must set commit_checkpoint_interval > 1;

Comment (WanYixian) [reply docs/guides/sink-to-clickhouse.md]: Updated
image

**Review by xxhZs — APPROVED** [review]
lgtm

**Pull request merged**